package.json

```diff
@@ -16,7 +16,7 @@
     "cc:slither:triage": "slither . --compile-force-framework hardhat --triage-mode",
     "cc:slither:pausable": "slither . --compile-force-framework hardhat --print not-pausable",
     "cc:node": "hardhat node --hostname 0.0.0.0",
-    "cc:compile": "hardhat compile && hardhat export --export-all contracts/deployments.json && ts-node --esm scripts/extract-abis.mts && wagmi generate",
```

Comment:
There was an error in the file extension. Can you revert its name to `.mts` and cancel any change you made to this file?